docs/SNMP_Alarms.md

```diff
@@ -44,15 +44,17 @@ The alarm models used by Clearwater are defined in [alarmdefinition.h](https://g
 
 ## Alarm Resiliency
 
-Clearwater is designed so that if an EMS receiving alarm notifications fails and
-recovers, it will learn about any alarms raised by Clearwater while it was down,
-rather than those alarms just being lost. This is due to the following two
-features:
-
-*   The process responsible for sending SNMP notifications, the Alarm Agent,
-    keeps track of all the currently active error states in a table called the
-    Alarm Active Table. Upon restart, an EMS can read this table and
-    learn about any SNMP INFORMs it may have missed in its downtime.
-*   If an EMS does not support reading the Active Alarm Table we can still
-    recover the SNMP INFORMs by running `/usr/share/clearwater/bin/sync_alarm.py`.
-    This will cause SNMP notifications to be resent to the EMS.
+Because prompt, accurate notifications of alarms are important for running a reliable telecoms network, Clearwater is designed so that if an EMS or an internal Clearwater component responsible for alarm notifications fails and recovers, the EMS will learn about any alarms raised by Clearwater during the outage, rather than those alarms just being lost. The exact behaviour is as follows:
```

Comment:
What would my year 9 English teacher say if he knew I was working with people who begin sentences with 'Because'?????????

He'd hate it.